modules/tinymce/src/themes/silver/test/ts/browser/editor/sizing/ResizeNotInRootTest.ts

```diff
@@ -0,0 +1,38 @@
+import { Assertions, Chain, Log, Pipeline, UiFinder } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyApis, TinyLoader } from '@ephox/mcagar';
+import { Insert, Remove, SugarBody, SugarElement, Width } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+
+import Theme from 'tinymce/themes/silver/Theme';
+
+UnitTest.asynctest('Editor fixed_toolbar_container resize test', (success, failure) => {
+  Theme();
+  const expectedWidth = 300;
```

Comment:
Maybe add some spacing here to make it more readable? The original version had spaces, so I suspect this is just a copy/paste issue.